internal/api/utils.go

```diff
@@ -232,19 +218,29 @@ func transformComponentUnits(recommendationJSON map[string]interface{}) map[stri
 		return value
 	}
 
-	convertCPU := func(cpu map[string]interface{}) error {
+	truncateCPU := func(cpu map[string]interface{}) bool {
 		cpuInCores, ok := cpu["amount"].(float64)
 		if ok {
-			if math.Abs(cpuInCores) < 1 {
-				cpuInMillicores := cpuInCores * 1000
-				cpu["amount"] = math.Round(cpuInMillicores) // millicore values are rounded & don't require decimal precision
-				cpu["format"] = "m"
-			} else {
-				cpu["amount"] = truncateToThreeDecimalPlaces(cpuInCores)
-				cpu["format"] = nil
-			}
+			cpu["amount"] = truncateToThreeDecimalPlaces(cpuInCores)
+			cpu["format"] = nil
 		}
-		return nil
+		return ok
+	}
+
+	truncateMemory := func(mem map[string]interface{}) bool {
+		memory, ok := mem["amount"].(float64)
+		if ok {
+			mem["amount"] = math.Trunc(memory*100) / 100
+		}
+		return ok
+	}
+
+	truncateDurationInHours := func(intervalData map[string]interface{}) bool {
+		durationInHours, ok := intervalData["duration_in_hours"].(float64)
+		if ok {
+			intervalData["duration_in_hours"] = math.Trunc(durationInHours*10) / 10
+		}
+		return ok
```

Comment:
Noticed values as `int` for short term and `float` for other using sample csv, same applies for values in openapi spec

Reply:
Updated the Openapi.json